modules/standard/Random.chpl

```diff
@@ -601,15 +841,15 @@ module Random {
       which has 64 bits of state and 32 bits of output.
 
       While the PCG RNG used here is believed to have good statistical
-      properties, it is not suitable for cryptographic use. Additionally,
-      if statistical properties of the random numbers are very important,
-      another strategy may be required.
+      properties, it is not suitable for generating key material for encryption.
```

Comment:
Given the explanation you gave in the PR conversation, I think this would be better if it gave the rationale (good cryptologists will know presumably, but so that the comment makes sense to the rest of us):  E.g., I'm imagining tacking on something like "due to the predictability (reproducibility?  determinism?) of the values generated by the stream."